src/main/java/no/nav/tag/kontakt/oss/fylkesinndelingMedNavEnheter/FylkesinndelingMedNavEnheter.java

```diff
@@ -0,0 +1,65 @@
+package no.nav.tag.kontakt.oss.fylkesinndelingMedNavEnheter;
+
+import lombok.NoArgsConstructor;
+
+import java.util.*;
+
+@NoArgsConstructor
+public class FylkesinndelingMedNavEnheter {
+
+    private Map<String, List<KommuneEllerBydel>> mapFraFylkesenheterTilKommunerOgBydeler;
+    private Map<NavEnhet, NavFylkesenhet> navEnhetTilFylkesenhet;
+    private Map<KommuneEllerBydel, NavEnhet> kommuneEllerBydelTilNavenhet;
+
```

Comment:
Denen klassen er litt rar. Den representerer selve retur-objektet, men hvis jeg har skjønt det riktig er det er bare den ene (øverste) av de tre medlemsvariablene som faktisk returneres, mens de to andre fungerer som parametre mellom de ulike metodene. Det kunne vært skrevet om slik at de to nederste sendes inn som "ekte" parametre til metodene der de benyttes. Mapping-logikken kunne antagelig blitt skrevet litt mer funksjonell generelt, og det er _mulig_ (men slett ikke sikkert) at det kunne blitt mer oversiktlig.

Reply:
Objektet brukes ikke som returverdi lenger; kan refaktorere koden senere hvis objektet fremdeles er rart